v2/tests/test_runner.py

```diff
@@ -3,18 +3,40 @@
 import runner
 
 
-def test_create_command() -> None:
+def test_create_command_complete() -> None:
     spec = runner._RunnerSpec(  # pylint: disable=protected-access
         python_executable=pathlib.Path("/usr/bin/python3"),
         robot_target=pathlib.Path("~/suite/calculator.robot"),
         outputdir=pathlib.Path("/tmp/outputdir/"),
-        output="output",
+        output=pathlib.Path("/tmp/outputdir/output.xml"),
+        previous_output=None,
         variablefile=None,
         argumentfile=None,
+        retry_strategy=runner._RetryStrategy.COMPLETE,  # pylint: disable=protected-access
```

Comment:
I don't think the _RetryStrategy belongs here, I would rather call this parameter rerun-failed, with rerun failed having time `Output` | None. The `Retrier` is responsible for building `Runners`. But since we are prototyping this is not a hard requirement. I want to do some cleaning up anyway.

Reply:
I agree, not ideal, can be improved later.